tests/integration_frameworks/llm/openai/test_openai_llmobs.py

```diff
@@ -70,12 +70,7 @@ def test_chat_completion(self, test_agent: TestAgentAPI, test_client: FrameworkT
             input_messages=[{"role": "user", "content": "Hello OpenAI!"}],
             output_messages=[{"role": "assistant", "content": "Hello! How can I assist you today?"}],
             metadata=expected_metadata,
-            metrics={
-                "input_tokens": mock.ANY,
-                "output_tokens": mock.ANY,
-                "total_tokens": mock.ANY,
-                "cache_read_input_tokens": mock.ANY,
-            },
+            metrics=mock.ANY,
         )
```

Comment:
i was thinking to be super specific with what we assert we could do something like

```suggestion
            metrics=mock.ANY,
)

assert llm_span_event["metrics"]["input_tokens"] = mock.ANY
assert llm_span_event["metrics"]["output_tokens"] = mock.ANY
assert llm_span_event["metrics"]["total_tokens"] = mock.ANY
```

just so we can still assert that at least those keys are there, as a regression test. this could apply to the other tests in this class as well